rest-service/manager_rest/deployment_update/handlers.py

```diff
@@ -838,5 +861,118 @@ def finalize(self, dep_update):
         self.sm.update(deployment)
 
 
+class DeploymentDependencies(UpdateHandler):
+    def _handle_dependency_changes(self,
+                                   dep_update,
+                                   query_filters,
+                                   dep_plan_filter_func,
+                                   keep_outdated_dependencies=False):
+        """Handles the dependency changes between what's currently in the DB
+        and what's in the new plan.
+
+        :param dep_update: a Deployment Update object.
+        :param query_filters: filters to use when querying about the current DB
+         dependencies. This function performs the changes on those queried
+         dependencies. These must match the filters arguments names of the
+         StorageManager.list(..) method.
+        :param dep_plan_filter_func: a function to filter the dependencies
+         registered in the deployment plan. The function gets a dependency
+         creator and outputs True if the dependency with that creator should
+         be handled.
+        :param keep_outdated_dependencies: if to remove dependencies that
+         aren't found in the new plan.
+        """
+        curr_dependencies = {
+            dependency.dependency_creator: dependency
+            for dependency in self.sm.list(
+                models.InterDeploymentDependencies,
+                get_all_results=True,
+                **query_filters)
+        }
+        new_dependencies = dep_update.deployment_plan.setdefault(
```

Comment:
It might be, but it has the same behavior, besides adding `'inter_deployment_functions': {}` to the `deployment_plan` in case this key doesn't exist. So I guess it's fine to leave it.